app/Http/Controllers/BeatmapsController.php

```diff
@@ -51,26 +51,30 @@ public function scores($id)
         }
 
         $beatmap = Beatmap::findOrFail($id);
+        $mode = Request::input('mode', Beatmap::modeStr($beatmap->playmode));
 
-        $scores = $beatmap
-            ->scoresBest()
-            ->defaultListing()
-            ->limit(config('osu.beatmaps.max-scores'))
-            ->with('user');
+        try {
```

Comment:
mostly because i wrote [this](https://github.com/ppy/osu-web/pull/536/files#diff-8145027d3f1be3158e29c97f85efac37R106) to throw an exception (but I think you may have noticed that already :stuck_out_tongue: )